apps/bixarena/tools/pyproject.toml

```diff
@@ -24,12 +24,17 @@ readme = 'README.md'
 dependencies = [
     "locust>=2.35.0",
     "numpy>=1.26.0",
+    "psycopg[binary]>=3.0.0",
+    "python-dotenv>=1.2.1",
     "rich>=13.7.0",
     "scipy>=1.11.0",
     "tqdm>=4.60.0",
-    "typer>=0.12.3"
+    "typer>=0.12.3",
 ]
 
+[project.scripts]
+bixarena = "bixarena_tools.main:app"
```

Comment:
We may want to standardize script names to align with project names in the future.